src/d3a/models/strategy/area_agents/two_sided_pay_as_clear_engine.py

```diff
@@ -127,12 +127,14 @@ def _match_offers_bids(self):
         clearing_rate, clearing_energy = clearing
         if clearing_energy > 0:
             self.owner.log.info(f"Market Clearing Rate: {clearing_rate} "
-                                f"||| Clearing Energy: {clearing_energy} ")
+                                f"||| Clearing Energy: {clearing_energy} "
+                                f"||| Market: {self.markets.source.area.name}")
```

Comment:
I guess, adding market's area name is redundant as _self.owner.log.info()_ already shows the area description at which the clearing is done.

Reply:
You are right, I will remove this, thanks

Reply:
On second thought this is not redundant, because these are 2 separate markets: the one that is reported on the info log output is the area name of the agent, whilst the market the log is referring to here is the market in which the clearing is performed. These 2 could be the same, but (depending on the order which the random function will call the tick method across IAA engines) 50% of the time it is not. Therefore since I find it useful to see in this log in which market did the clearing happen, I am voting to keep this log.

Reply:
Thanks for the explanation. I would then vouch to simply use _log.warn_ rather than _self.owner.log_